### PR TITLE
Added Guess My Number game

### PR DESCRIPTION
**Pull request by swarajzz**

### CODE REVIEW

File: public/Projects/SwarajGuessMyNumber/info.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    name: "Swaraj",
+    ghurl: "https://github.com/swarajzz",
+    imgName: "GuessMyNumber.gif",
```
Comment (SandeepKrSuman) [inline public/Projects/SwarajGuessMyNumber/info.txt:4]: yours is a `.png` file not a `gif`
so it should be 
```
imgName: "GuessMyNumber.png",
```

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]
Just do the mentioned change inside `info.txt`
everything else looks fine 👍

**Review by SandeepKrSuman — APPROVED** [review]
LGTM 🚀

**Comment by swarajzz** [comment]
Thank you

**Pull request merged**